daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -105,6 +105,19 @@ The following table lists the properties for metrics:
 |--------------|--------|-------------|
 | `enabled` | boolean | Whether metrics should to be enabled.
 
+To mitigate high memory usage and egress costs associated with [high cardinality metrics]({{< ref "metrics-overview.md#high-cardinality-metrics" >}}), you can set regular expressions for every metric exposed by the Dapr sidecar. For example:
+
+```yml
+metric:
+    enabled: true
+    rules:
```

Comment:
Can you add rules into the table above. Like this

| Property     | Type   | Description |
|--------------|--------|-------------|
| `rules` | boolean | Named rule to filter metrics. Each rule contains a set of `labels` to filter on and `regex`expression to apply to the metrics path.